graphwalker-io/src/main/java/org/graphwalker/io/factory/dot/DotContextFactory.java

```diff
@@ -190,11 +190,8 @@ public String getAsString(List<Context> contexts) {
   public void write(List<Context> contexts, Path path) throws IOException {
     File folder = path.toFile().getAbsoluteFile();
     Path dotFile = Paths.get(folder.toString(), contexts.get(0).getModel().getName() + ".dot");
-    OutputStream outputStream = Files.newOutputStream(dotFile);
-    try {
+    try (OutputStream outputStream = Files.newOutputStream(dotFile)) {
       outputStream.write(String.valueOf(getAsString(contexts)).getBytes());
-    } finally {
-      outputStream.close();
```

Comment:
try-with-resource should handle closing of the stream, is that an old coverity report? or is it for this branch?